### ISSUE TITLE
Missing NPC conversations

### ISSUE THREAD

**Issue by Porkechebure** [d07RiV/diabloweb#83]
I think it's still related to these (I play the game [here](https://d07riv.github.io/diabloweb/*)
[Issue 62](https://github.com/d07RiV/diabloweb/issues/62)
[issue 223](https://github.com/diasurgical/devilutionX/issues/223)

I get all the NPCs talking about Gossip (which changes and it's not always the same talk), The Butcher, Poisoned Water supply quests. I am at dungeon lvls 12-13 and there should already have been a talk about Sin War, the [Arkaine's Valor Armour](https://d07riv.github.io/diabloweb/).

Until now I get only the 3 aforementioned NPC talks

**Comment by d07RiV** [comment]
Yep I saw the same problem in devilution, haven't dug into the code yet to look for the cause, though.

**Comment by Porkechebure** [comment]
> Yep I saw the same problem in devilution, haven't dug into the code yet to look for the cause, though.

Just a question, what is devilution? What's different about this?
From what I get: 
devilution-> reverse engineering of the game
diabloweb-> uses devilution to run diablo on web through webassembly

Is this correct?

**Comment by ericdrobinson** [comment]
>I get all the NPCs talking about Gossip (which changes and it's not always the same talk)

Same here. I've only played through 4 levels of the labyrinth but I've never seen anything besides "Gossip" in the NPC chats...

**Comment by AJenbo** [comment]
This issue has now been solved upstream: https://github.com/diasurgical/devilution/commit/1c9801986c559417b7889708b2ce9cc2e27d9ba9


> devilution-> reverse engineering of the game
> diabloweb-> uses devilution to run diablo on web through webassembly

Correct

**Comment by ericdrobinson** [comment]
Can't wait until the fix makes its way into `diabloweb`!

**Comment by ericdrobinson** [comment]
@d07RiV [According to](https://github.com/d07RiV/diabloweb/issues/83#issuecomment-532019806) @AJenbo, this has been fixed upstream. Any update on when this might make its way into a new diabloweb release?
